app.py

```diff
@@ -67,33 +71,42 @@ def register():
             flash("Username already exists")
             return render_template("register.html")
 
+@app.route("/profile", methods=["GET"])
+def profile():
+    return render_template("profile.html")
 
 @app.route("/logout", methods = ["GET","POST"])
 def logout():
-        # remove the username from te session if it's there
+        # remove the username from the session if it's there
         session.pop('user', None)
         flash('You were successfully logged out!')
         return redirect(url_for('login'))
 
+@app.route("/settings", methods = ["GET", "POST"])
+def settings():
+    return render_template("settings.html")

```

Comment:
settings requires a password change form on the frontend and a function to actually change on the backend 
make sure to check if method is get or post